pkg/config/default_windows.go

```diff
@@ -5,7 +5,7 @@ import "os"
 // getDefaultImage returns the default machine image stream
 // On Windows this refers to the Fedora major release number
 func getDefaultMachineImage() string {
-	return "35"
+	return "testing"
 }
```

Comment:
if all providers should return the same thing then this function should be removed from the `_<os>.go` files and the default should just be dinfed once for eveything

Also the comment of this function is now wrong.